docs/Concepts/Stores.md

```diff
@@ -16,6 +16,10 @@ QKM defines the following store interfaces:
 - [Key store](#key-store)
 - [Ethereum store](#ethereum-store)
 
+## Vault
+
+A vault defines the user credentials required to access to the remote storage, such as HashiCorp Vault, Azure Key Vault, or AWS KMS.
```

Comment:
I think we use "secure storage system" instead of "remote storage"

Reply:
Updated